src/main/java/org/mitre/synthea/world/agents/behaviors/planfinder/IPlanFinder.java

```diff
@@ -42,8 +42,7 @@ public static boolean meetsAffordabilityRequirements(
     // Occupation determines whether their employer will pay for insurance after the mandate.
     double occupation = (Double) person.attributes.get(Person.OCCUPATION_LEVEL);
 
-    return plan.accepts(person, time)
```

Comment:
Maybe answering my own question, it looks like `accepts` is now used in `PlanFinderPriority.find` so only plans for which the bene is eligible will be include?

Reply:
Yup, I did this because I didn't think it made sense for a method called "meetsAffordabiltyRequirements" to make a check for whether a plan would accept a person. The `accepts` check got moved up in the hierarchy, and this ended up also improving some of the logic when filtering eligible plans, as you noted.